client/my-sites/sidebar/site-menu.jsx

```diff
@@ -107,6 +109,7 @@ class SiteMenu extends PureComponent {
 			{
 				name: 'comments',
 				label: translate( 'Comments' ),
+				count: get( this.props.commentCounts, 'pending' ),
```

Comment:
Minor: can we now use the conditional chaining operator to avoid needing lodash? That said, if Calypso has a convention on `get()` then ignore me.

Reply:
I have copied this from some other place where they were reading comment counts. I'd rather keep it identical and then refactor at once.